### ISSUE TITLE
[Bug]: Previously dragging a non-md file onto board displayed a link to the file name in the card - it now attempts to renders the file

### ISSUE THREAD

**Issue by phil31753** [mgmeyers/obsidian-kanban#970]
### Describe the bug

I am using Markdown links in my vault.

![image](https://github.com/mgmeyers/obsidian-kanban/assets/95326157/39901bc4-295e-4f66-9bef-bc4f484ddb72)

Previously, when dragging a non-markdown file (.docx, .pptx, .pdf, image files...) onto the Knaban board the file name was displayed. Clicking on the filename opened the file externally to Onsidian with the associated app.

This was a useful way to attach external documents to a Kanban board.

Since the last release, the Kanban plugin attempts to render the file on the card. For .pdf files the heavily cropped top right corner of the file is rendered. Image files are rendered as a thumbnails. For other files, the rendered version of the filename is displayed.

![image](https://github.com/mgmeyers/obsidian-kanban/assets/95326157/5e9259bc-1740-4fd0-8141-ed69c78c1099)

### Expected Behavior

When a non-markdown file is added to a Kanban board, the plugin should revert to previous behavior and display the filename as a link that can be clicked on to launch the file in an external app.

### Steps to reproduce

1. Drag a non-markdown file to the Kanban board.
2. Confirm that a rendered version of the file is displayed.

### Operating system

Windows

**Comment by jowibou** [comment]
Yes this is quite annoying. Dragging a pdf, for instance, onto a kanban sticks a ! before the link, and thus tries to embed it, forcing us to edit the entry.
